packages/listview/src/components/FloatingSearch/__test__/floatingSearch.spec.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import Search from '../search';
+
+describe('<FloatingSearch />', () => {
+    it('should render an input type search and a button', () => {
+        const component = mount(<Search />);
+        const input = component.find('input');
+        const button = component.find('button');
```

Comment:
Here remember don't do introspection, unless it is necessary, always assume that you don't know what is inside a component. In this case what you know is that there is an Input and ButtonIcon component, not input and button tags